test/getGrist.ts

```diff
@@ -117,7 +117,8 @@ export class GristTestServer {
 
   public get assetUrl() {
     const {contentPort} = serverSettings;
-    return `http://localhost:${contentPort}`;
+    // localhost doesn't work on Node 18 (see https://github.com/node-fetch/node-fetch/issues/1624)
```

Comment:
That's a bit scary. I see it is ipv6 related - https://github.com/node-fetch/node-fetch/issues/1624#issuecomment-1407717012

Does this have any consequences for the move to node 18 in Grist? I feel like there's a lot of dependence on localhost in test machinery?

Reply:
I don't think we make any outbound requests to `localhost` from Node, and the failures I've seen since the upgrade feel more symptomatic of general flakiness.

I see one option is to do `dns.setDefaultResultOrder('ipv4first')` to go back to the pre-17 behavior.

Reply:
Yeah. I suspect that all may still be working. Even in this file, the fetch to localhost works fine for the Grist instance running in Docker - it's just live-server that is having the ipv6 problem. We should take a closer look after.